src/lib.rs

```diff
@@ -51,3 +53,124 @@ pub use telio_lana;
 
 /// cbindgen:ignore
 pub use telio_firewall;
+
+pub use uniffi_libtelio::*;
+#[allow(clippy::panic, clippy::unwrap_used, clippy::expect_used, unwrap_check)]
+mod uniffi_libtelio {
+    use std::net::{IpAddr, SocketAddr};
+
+    use ipnetwork::IpNetwork;
+
+    use super::crypto::{PublicKey, SecretKey};
+    use super::*;
+
+    use telio_model::api_config::*;
+    use telio_model::config::*;
+    use telio_model::mesh::*;
+    use telio_utils::{Hidden, HiddenString};
+
+    impl UniffiCustomTypeConverter for PublicKey {
+        type Builtin = String;
+
+        fn into_custom(val: Self::Builtin) -> uniffi::Result<Self> {
+            Ok(val.parse().map_err(|_| TelioError::InvalidKey)?)
+        }
+
+        fn from_custom(obj: Self) -> Self::Builtin {
+            String::from_utf8_lossy(&obj.0).to_string()
```

Comment:
I don't think this is correct, why not instead convert it to something like base64?

Reply:
I updated these to use `Vec<u8>`instead of `String` since that is what the keys use under the hood. I think I remember getting some errors when I tried to do that in the past, but it worked now